anomaly_detector/events/anomaly_event.py

```diff
@@ -0,0 +1,34 @@
+import requests
+class AnomalyEvent():
+
+    def __init__(self, predict_id, message, score, anomaly_status):
+        self.predict_id = str(predict_id)
+        self.message = message
+        self.score = score
+        self.anomaly_status = anomaly_status
+
+    def isEvtFalseAnomaly(self):
+        """
+        Process Anomaly Event
+
+        Stores anomaly into factstore and validates if this anomaly was previous reported as false.
+        It will also return a message with  for example 'status: false'
+        Also this
+
+        :param predict_id: This id is autogenerated and used to track events of anomaly.
+        :param message: the message that was processed via logs stream
+        :param score: calculated via machine learning algorithm
+        :param anomaly_status: when the score is greater then the threshold.
+        :return: will return 1 if it was a true anomaly.  0 if it was a false anomaly.
+        """
+        p_id= str(self.predict_id)
+        headers = {"X-LAD-APP-ID","999"} # TODO: Set this as an env var
```

Comment:
I'm assuming we may have more teams using this service so I think we should have this idea of app-id for us to be able to track systems separately. As we generate metadata files 1 for events and 1 for feedback.